tests/test_tpt_reports.py

```diff
@@ -135,3 +135,29 @@ def test_domain_validation():
         except SystemExit as sys_exit:
             return_code = sys_exit.code
             assert return_code == 2, "main() should return with error return code 2"
+
+
+def test_parse_json():
+    """Validate parse json functionality."""
+    payloads_list = []
+    payloads_meta = {}
+    payload_list_test = [
+        {
+            "Payload": "Test payload",
+            "C2 Protocol": "c2_1",
+            "Border Protection": "Not Blocked",
+            "Host Protection": "Blocked",
+        }
+    ]
+    data = tpt_reports.tpt_reports.load_json_file("tests/data/test.json")
```

Comment:
It doesn't totally matter now, but if more tests are going to be using `tests/data/test.json`, then it would make sense to make it a global constant e.g.:
```
TEST_JSON_FILE = "tests/data/test.json"
```

Feel free to take or leave this suggestion, just let me know what you decide and I will resolve this conversation.

Reply:
I see you did this in https://github.com/cisagov/tpt-reports/pull/52/commits/49e9779bed2c2275d3a86ae2a7d5e7c08d889b55 👍

Reply:
Resolved this in this commit 49e9779bed2c2275d3a86ae2a7d5e7c08d889b55